applications/onchainvision.md

```diff
@@ -0,0 +1,132 @@
+
+#### Enhancing Reliability and Accessibility
+
+By eliminating reliance on external services, we ensures the long-term reliability and accessibility of digital assets. No longer will assets be at risk of becoming inaccessible due to external service downtime or discontinuation. This reliability is a cornerstone of trust in the blockchain token ecosystem.
+
+#### Unlocking New Monetization Opportunities
+
+The programmability of OnChainVision's digital assets opens doors to innovative monetization strategies. These smart contracts can be designed to facilitate revenue-sharing models, enable referral programs, or even allow users to license their assets for use in various applications. The possibilities for generating income from your digital assets are limited only by your imagination!
+
+## Team
+
+### Team members
+
+- Herald Bayoca - Head of Research and Development
+
+### Team Website
+
+- [OnChainVision Website](https://onchainvision.raldblox.com)
+
+### Team's experience
+
+- Herald Bayoca is an independent researcher, developer and a proactive problem-solver that strategically blend his passion and creativity with his technical expertise. Experienced in machine design, machine learning, space tech, fullstack web development, enterprise blockchain, post-quantum cryptography and quantum computing.
+- His Projects and Awards:
+  - [FileBlox](https://github.com/raldblox/fileblox) - an encrypted NFT Marketplace that solves the right-click-and-save problem for content creators
+    - Won Two Awards at [EthGlobal HackFS 2023](https://ethglobal.com/showcase/fileblox-y0rjm)
+      - Best Use of Interplantery File System (IPFS)
+      - Best ApeCoin Integration
+  - [Namespace](https://github.com/raldblox/namespace) - a next-generation decentralized naming and identity system revolutionizing
+    - Won Second Place at [Patex Global Hackathon](https://dorahacks.io/buidl/5728?badge=b6330)
+    - Received Grants from [AptosDAO Round3](https://dorahacks.io/aptos/round-3/buidl)
+  - [TokenTreats](https://github.com/raldblox/tokentreats) - a cheat-proof loyalty reward platform with personalized blockchain tokens
+    - Won WorldCoin Prize Pool at [EthGlobal Superhack 2023](https://ethglobal.com/showcase/tokentreats-ihi4a)
+
+### Team Code Repos
+
+- [Front End - Private](https://github.com/raldblox/onchainvision)
+- [Packages, Contracts, and Libraries - OpenSource](https://github.com/onchainvision-labs/onchainvision-contracts)
+
+### Team LinkedIn Profiles
+
+- [Herald Bayoca](https://www.linkedin.com/in/rald)
+
+## Development Roadmap
+
+### Roadmap for OnChainVision
+
```

Comment:
https://github.com/vechain/grant-program/blob/master/applications/application-template.md#overview-1 We have a rule that limits milestone 1 funds should not be higher than 40% of the total amount you are applying for, I would suggest adding more milestones.